client/lib/abtest/active-tests.js

```diff
@@ -104,5 +104,13 @@ module.exports = {
 		defaultVariation: 'showMonthly',
 		allowExistingUsers: true
 	},
-
+	signupDomainsHeadline: {
+		datestamp: '20170306',
+		variations: {
+			original: 50,
+			updated: 50
+		},
+		defaultVariation: 'original',
+		allowExistingUsers: true
```

Comment:
@lucasartoni Should we allow existing users to participate in the test (as it is now) or not?